api/types/authority.go

```diff
@@ -89,9 +83,9 @@ type CertAuthority interface {
 	// SetRotation sets rotation state.
 	SetRotation(Rotation)
 	// GetSigningAlg returns the signing algorithm used by signing keys.
-	GetSigningAlg() string
+	GetSigningAlg() CertAuthoritySpecV2_SigningAlgType
```

Comment:
Why was the return type changed here?

Reply:
To make `GetSigningAlg` a simple getter method, and remove the dependency on `golang.org/x/crypto/ssh`, which is used to convert the CertAuthoritySpecV2_SigningAlgType into a string. That functionality was moved to `sshutils.GetSigningAlg`.